.devcontainer/Dockerfile.base

```diff
@@ -12,13 +12,28 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM mcr.microsoft.com/devcontainers/rust:1-1-bookworm
+FROM mcr.microsoft.com/vscode/devcontainers/base:1-debian-12 AS src
+
+FROM ubuntu:24.04
 
-ARG USERNAME=vscode
 ARG TARGETARCH
+ARG USERNAME=vscode
 
-RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get -y install \
+# All the packages we need
+RUN export DEBIAN_FRONTEND=noninteractive \
+    && apt update && \
+    if [ "$TARGETARCH" = "amd64" ]; then \
+        apt install -y gcc-aarch64-linux-gnu; \
+    elif [ "$TARGETARCH" = "arm64" ]; then \
+        apt install -y gcc-x86-64-linux-gnu; \
+    fi \
+    && apt -y install \
+    # Basics
+    git \
+    zsh \
+    curl \
+    iputils-ping \
```

Comment:
Do we really need this package?

Reply:
You mean `iputils-ping`? Me and @krucod3 were missing that in the previous dev container. It provides the ping command which is required to test various connections problems. The size of the corresponding .deb file is only 43 kB so it should be OK.